### ISSUE TITLE
Median unreliable

### ISSUE THREAD

**Issue by cookpa** [ftdc-picsl/QuANTs#16]
As discovered by @caugolm 

The median computed by

https://simpleitk.org/doxygen/v2_4/html/classitk_1_1simple_1_1LabelIntensityStatisticsImageFilter.html

Is based on a histogram calculation. The default number of bins is 128, over the whole intensity range of the image. This leads to bad median calculation when the values are tightly clustered, and is susceptible to outliers in the image.

Increasing the number of histogram bins to 10000 is an immediate fix, but IMO better to avoid this danger by using numpy to get the median.

**Comment by jeffduda** [comment]
I agree wih the switch to numpy. This would be a good time to add a new column to the output for 'calculator' to indicate what package was used to calculate a given value. While it's possibly overkill for just this one change, it will make it easier to include values from additional packages like pyradiomics
